webpack.mix.js

```diff
@@ -75,7 +75,8 @@ mix.copy('node_modules/bootstrap-datepicker/dist', 'src/public/packages/bootstra
 	.copy('node_modules/places.js/dist', 'src/public/packages/places.js/dist')
 	.copy('node_modules/jquery-ui-touch-punch/jquery.ui.touch-punch.min.js', 'src/public/packages/jquery-ui-touch-punch/jquery.ui.touch-punch.min.js')
     .copy('node_modules/urijs/src/URI.min.js', 'src/public/packages/URI.js/URI.min.js')
-    .copy('node_modules/intl-tel-input/build', 'src/public/packages/intl-tel-input/build');
+    .copy('node_modules/intl-tel-input/build', 'src/public/packages/intl-tel-input/build')
+    .copy('node_modules/dropzone/dist', 'src/public/packages/dropzone/dist');
```

Comment:
There should also be a `package.json` change here - dropzone becomes a dependency if we do `npm install dropzone` - https://github.com/Laravel-Backpack/CRUD/blob/main/package.json

Reply:
Done